docs/literate.jl

```diff
@@ -45,6 +53,8 @@ function preprocess(content)
 # [Julia source file](@__REPO_ROOT_URL__/examples/@__NAME__/script.jl).
 #md # The corresponding notebook can be viewed in [nbviewer](@__NBVIEWER_ROOT_URL__/examples/@__NAME__.ipynb).*
 #nb # The rendered HTML can be viewed [in the docs](https://juliagaussianprocesses.github.io/ApproximateGPs.jl/dev/examples/@__NAME__/).*
+#
+# ---
```

Comment:
It looks like this breaks the formatting in [nbviewer](https://nbviewer.org/github/JuliaGaussianProcesses/ApproximateGPs.jl/blob/gh-pages/previews/PR104/examples/a-regression.ipynb) but not when you [view the notebook directly in GH](https://github.com/JuliaGaussianProcesses/ApproximateGPs.jl/blob/gh-pages/previews/PR104/examples/a-regression.ipynb) for some reason?

Reply:
Strange that nbviewer seems to struggle with... viewing notebooks

Reply:
seems fixed now